hamburgerHW24/index.js

```diff
@@ -21,19 +21,19 @@ class Hamburger {
     calculateCalories() {
         this.calories = 0;
 
-        if(this.size === "small")
+        if(this.size === Hamburger.SIZE_SMALL)
```

Comment:
Let's refactor and move some code to separate methods
For calories

```
getCalories(type) {
  switch(type) {
    case Hamburger.SIZE_SMALL:
      return 20;
    case ...:
      return n;
      // ...
    default:
      return 0;  
  }
}
```

Then use it in the calculation

```
this.calories += this.getCalories(this.size);
```


The same you can apply for prices